### ISSUE TITLE
Create a basic form component for new scans.

### ISSUE THREAD

**Issue by stephane-segning** [stephane-segning/lynx-scanner#9]

**Comment by Christiantyemele** [comment]
Hello @stephane-segning I propose using formik library for this ticket

**Comment by stephane-segning** [comment]
> Hello @stephane-segning I propose using formik library for easy form handling and. error catching in this ticket

Yes, and this is only for the form. This is exactly the purpose of this ticket.
